libs/components/packages/src/schematics/migrations/prettier-ignore-assets/prettier-ignore-assets.schematic.ts

```diff
@@ -0,0 +1,29 @@
+import { Rule } from '@angular-devkit/schematics';
+
+export default function addAssetsToPrettierIgnore(): Rule {
+  return (tree) => {
+    const spaAssetsPath = 'src/assets';
+    const libraryAssetsPath = 'projects/*/src/assets';
```

Comment:
It'd be nice to be able to add more paths in the future as needed; maybe an array of ignored paths would be appropriate? I'd also start the paths with a root-relative `/` so we're specific about which paths we're ignoring.
```
const ignorePaths = ['/.angular/cache', '/projects/*/src/assets', '/src/assets'];
for (const ignorePath of ignorePaths) {...}
```